### PR TITLE
Fix student bugs

### PR DESCRIPTION
**Pull request by LimAiLin**

### CODE REVIEW

File: src/main/java/seedu/address/model/student/Attendance.java [hunk 1]
```diff
@@ -11,7 +11,7 @@ public class Attendance {
 
     public static final String MESSAGE_CONSTRAINTS =
             "Attendance should only contain numbers";
```
Comment (heyzec) [inline src/main/java/seedu/address/model/student/Attendance.java:13]: Can also change this to "only contain nonnegative whole numbers"

File: src/main/java/seedu/address/logic/parser/student/StudentParserUtil.java [hunk 2]
```diff
@@ -166,6 +166,7 @@ public static Attendance parseAttendance(String attendance) throws ParseExceptio
         if (!Attendance.isValidAttendance(trimmedAttendance)) {
             throw new ParseException(Attendance.MESSAGE_CONSTRAINTS);
         }
+        trimmedAttendance = trimmedAttendance.replaceFirst("^0+(?!$)", ""); //remove leading zeros
```
[the remark this answers is not in the document]
Comment (LimAiLin) [reply src/main/java/seedu/address/logic/parser/student/StudentParserUtil.java]: I agree its better stored as numbers to accommodate for features next time like incrementing the attendance and participation scores. I changed it do take a look.

File: src/main/java/seedu/address/model/student/Attendance.java (hunk 1 above)
Comment (LimAiLin) [reply src/main/java/seedu/address/model/student/Attendance.java]: Done

**Review by yeoyujie — APPROVED** [review]
LGTM

**Pull request merged**